atom/browser/native_window_views_win.cc

```diff
@@ -299,6 +300,68 @@ bool NativeWindowViews::PreHandleMSG(UINT message,
   }
 }
 
+void NativeWindowViews::HandleSizingEvent(WPARAM w_param, LPARAM l_param) {
+  double aspect_ratio = GetAspectRatio();
+  // aspect_ratio width/height
+  if (aspect_ratio == 0) {
```

Comment:
Testing a double for equality to an int zero is risky. Might be better to use the fabs idiom here.

```suggestion
  if (fabs(aspect_ratio) < 0.0001) {
```